### DISCUSSION TITLE
How to turn of line numbers?

### DISCUSSION THREAD

**Discussion by idr4n** [shortcuts/no-neck-pain.nvim#224] [Q&A] [answered]
Hi and thanks for a very nice plugin!

I was trying to turn off line numbers with some of the options but I couldn't. So far I have (using lazy.nvim):

```lua
{
    "shortcuts/no-neck-pain.nvim",
    cmd = { "NoNeckPain" },
    keys = {
        { "<leader>tz", "<cmd>NoNeckPain<cr>", desc = "Zen-mode (No-neck-pain)" },
    },
    opts = {
        buffers = { 
            wo = { 
                number = false,
                relativenumber = false
            },
         },
    },
},
```

I could always use the following keybinding though for this, but I though it might be possible to do it using the plugin's options:

```lua
        keys = {
            {
                "<leader>tz",
                function()
                    vim.cmd([[
                        NoNeckPain
                        set invnumber
                        set invrelativenumber
                    ]])
                end,
                desc = "Zen-mode (No-neck-pain)",
            },
        },
```

Thanks in advance!

**Comment by shortcuts** [comment, marked as answer]
Hey!

> I was trying to turn off line numbers with some of the options but I couldn't. So far I have (using lazy.nvim):

They are off by default, but your solution is correct, any window/buffer option you provide will be set to the NNP side windows.

If you want to make this effective to any opened window, you indeed have to define it globally, this plugin only alters the window it creates.
